projects/plugins/jetpack/extensions/shared/components/product-management-controls/toolbar-control.js

```diff
@@ -20,6 +20,15 @@ import useOpenBlockSidebar from './use-open-block-sidebar';
 import { getMessageByProductType } from './utils';
 import { store as membershipProductsStore } from '../../../store/membership-products';
 
+/**
+ * Check if it's in the context of the customizer.
+ *
+ * @returns {boolean} if we are in the context of Customizer.
+ */
+function isInCustomizer() {
+	return 'function' === typeof window?.wp?.customize;
+}
+
```

Comment:
Change this check with getEditorType calls. We are trying to centralize all the editor type checks and avoid duplicating knowledge of how we determine the context. https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/extensions/shared/get-editor-type.js